01factorial.js

```diff
@@ -0,0 +1,16 @@
+//input: an integer
+//output: integer representing the multiplication of n * n-1 * ... 1
+//constraints: must be positive
+//edge cases: no
+
+var total = 1;
+var factorial = function (n) {
+  if (n === 1 || n === 0) {
```

Comment:
n === 0 is never executed. unless input can be 0. If you really want to be safe do n <=1